addons/account/tests/test_sequence_mixin.py

```diff
@@ -234,6 +234,20 @@ def test_sequence_draft_change_date(self):
             self.assertMoveName(new_multiple_move_1, 'AJ/15-16/01/0001')
             move_form.date = fields.Date.to_date('2016-01-10')
 
+    def test_sequence_draft_change_date_with_new_sequence(self):
+        invoice_1 = self.test_move.copy({'date': '2016-02-01', 'journal_id': self.company_data['default_journal_sale'].id})
+        invoice_2 = invoice_1.copy({'date': '2016-02-02'})
+        invoices = invoice_1 + invoice_2
+
+        self.assertMoveName(invoice_2, 'INV/15-16/0001')
+
+        resequence_wizard = Form(self.env['account.resequence.wizard'].with_context(active_ids=invoices.ids, active_model='account.move'))
+        resequence_wizard.first_name = "INV/15-16/02/001"
+        resequence_wizard.save().resequence()
+
+        invoice_2.date = fields.Date.to_date('2016-03-01')
+        self.assertMoveName(invoice_2, 'INV/15-16/03/001')
```

Comment:
You should be able to just set the name of `invoice_1` and assert the name/placeholder of `invoice_2` right after.
No need for the resequence wizard

```suggestion
        invoice_1.name = "INV/15-16/02/001"
        self.assertMoveName(invoice_2, 'INV/15-16/02/002')

        invoice_2.date = '2016-03-01'
        self.assertMoveName(invoice_2, 'INV/15-16/03/001')
```